Resources/public/js/views/ez-dashboardblocksview.js

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblocksview', function (Y) {
+    'use strict';
+    /**
+     * Provides the Dashboard Blocks View class
+     *
+     * @module ez-dashboardblocksview
+     */
+    Y.namespace('eZ');
+
+    var SELECTOR_CONTENT = '.ez-dashboard-content';
+
+    /**
+     * The dashboard blocks view
+     *
+     * @namespace eZ
+     * @class DashboardBlocksView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.DashboardBlocksView = Y.Base.create('dashboardBlocksView', Y.eZ.TemplateBasedView, [], {
+        initializer: function () {
+            this.onceAfter('activeChange', function (event) {
+                this.after('blocksChange', this._renderBlocks, this);
```

Comment:
as I was writing in https://github.com/ezsystems/PlatformUIBundle/pull/584#discussion_r62852782, please remove that, for now we'll only cover the case where the block are added before rendering. We'll see later if we need more than that.